client/my-sites/themes/activation-modal.jsx

```diff
@@ -124,13 +107,17 @@ export class ActivationModal extends Component {
 				<div className="themes__theme-preview-wrapper">
 					<h1 className="activation-modal__heading">
 						{ translate( 'Activate %(themeName)s', {
-							args: { themeName },
+							args: { themeName: newTheme.name },
 						} ) }
 					</h1>
 					<p className="activation-modal__description">
 						{ translate(
-							'After activation, this layout will replace your existing homepage. But you can still access your old content. {{a}}Learn more{{/a}}.',
+							'You’re about to change your active theme from {{strong}}%(activeThemeName)s{{/strong}} to {{strong}}%(newThemeName)s{{/strong}}.{{br}}{{/br}}{{br}}{{/br}}This will replace your homepage. Don’t worry, your old content can still be accessed. {{a}}Learn more{{/a}}.',
```

Comment:
What do you think about the ideas behind the following copy variations? cc: @michaelpick 

I got inspired by the copy in the section "What Happens to Your Old Content" in [Learn more](https://wordpress.com/support/themes/changing-themes/#what-happens-to-your-old-content). 

### **In this PR: "Don’t worry, your old content can still be accessed."**

<img width="670" alt="Screenshot 2566-10-19 at 13 20 56" src="https://github.com/Automattic/wp-calypso/assets/1881481/69521bf0-687f-4caf-a1fe-774a238fa7e7">


### **Suggestion 1: "You can recover it later."**

<img width="627" alt="Screenshot 2566-10-19 at 13 08 06" src="https://github.com/Automattic/wp-calypso/assets/1881481/c86ffc5d-fc63-406a-bffa-7548a0e586d7">

### **Suggestion 2: "Your old content remains on the site and you can recover it."**

<img width="663" alt="Screenshot 2566-10-19 at 13 10 08" src="https://github.com/Automattic/wp-calypso/assets/1881481/e9369a86-5d61-4fb9-a170-4ede0b238d1f">

### **Suggestion 3: "If you wish to recover it, reactivate Fewer to view and copy any of that content."**

<img width="665" alt="Screenshot 2566-10-19 at 13 14 24" src="https://github.com/Automattic/wp-calypso/assets/1881481/2dfc40e1-dd56-4646-9e34-2aaf877b2604">

Reply:
Thanks for the great suggestions! I went ahead with suggestion 2 with a small tweak:

```
Your old content remains on the site and can be recovered later.
```